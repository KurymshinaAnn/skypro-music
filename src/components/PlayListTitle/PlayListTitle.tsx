import classNames from "classnames";
import styles from "./PlayListTitle.module.css";

export default function PlaylistTitle() {
  return (
    <>
      <div className={styles.contentTitle}>
        <div className={classNames(styles.playlistTitleCol, styles.col01)}>
          Трек
        </div>
        <div className={classNames(styles.playlistTitleCol, styles.col02)}>
          Исполнитель
        </div>
        <div className={classNames(styles.playlistTitleCol, styles.col03)}>
          Альбом
        </div>
        <div className={classNames(styles.playlistTitleCol, styles.col04)}>
          <svg className={styles.playlistTitleSvg}>
            <use href="img/icon/sprite.svg#icon-watch"></use>
          </svg>
        </div>
      </div>
    </>
  );
}
