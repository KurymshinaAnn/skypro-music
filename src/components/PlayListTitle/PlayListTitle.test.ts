import renderer from "react-test-renderer";
import PlayListTitle from "./PlayListTitle";

it("Renders playlist title", () => {
  const playlistTitle = renderer.create(PlayListTitle());
  let tree = playlistTitle.toJSON();
  expect(tree).toMatchSnapshot();
});
