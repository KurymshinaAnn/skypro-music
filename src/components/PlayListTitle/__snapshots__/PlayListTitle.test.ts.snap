// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders playlist title 1`] = `
<div
  className="contentTitle"
>
  <div
    className="playlistTitleCol col01"
  >
    Трек
  </div>
  <div
    className="playlistTitleCol col02"
  >
    Исполнитель
  </div>
  <div
    className="playlistTitleCol col03"
  >
    Альбом
  </div>
  <div
    className="playlistTitleCol col04"
  >
    <svg
      className="playlistTitleSvg"
    >
      <use
        href="img/icon/sprite.svg#icon-watch"
      />
    </svg>
  </div>
</div>
`;
